public/js/gulliver.es6.js

```diff
@@ -278,13 +278,38 @@ function setupConfig() {
   }
 }
 
+function setupMessaging() {
+  const messaging = new Messaging();
+  messaging.init();
+
+  const checkbox = document.getElementById('notifications');
```

Comment:
It might make sense to encapsulate the notification element handler into a separate class (maybe even file).